src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -477,6 +477,11 @@ let setup_daemon logger =
          for the associated private key that is being tracked by this daemon. \
          You cannot provide both `uptime-submitter-key` and \
          `uptime-submitter-pubkey`."
+  and verify_local_snark_works =
+    flag "--verify_local_snark_works" no_arg
```

Comment:
This flag name isn't following the standard pattern for the CLI. The standard is to use `kebab-case` rather than `snake_case`.